modules/raft/src/main/java/org/apache/ignite/raft/jraft/core/DefaultJRaftServiceFactory.java

```diff
@@ -53,4 +49,12 @@ public DefaultJRaftServiceFactory(LogStorageFactory factory) {
         Requires.requireTrue(!StringUtils.isBlank(uri), "Blank raft meta storage uri.");
         return new LocalRaftMetaStorage(uri, raftOptions);
     }
+
+    @Override public LogEntryCodecFactory createLogEntryCodecFactory() {
```

Comment:
Override on the same line

Reply:
Not this time, I'm not gonna touch JRaft code.